### PR TITLE
Restructure; Refer to more services

### PR DESCRIPTION
**Pull request by GladOSkar**
See title.

Does this look OK to you? i figured structuring the translations like this was more usable.

The new french translations are generated by deepl.io so if you would accept this code i would request a review from the mailing list before merging.

Cheers :)

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -232,117 +232,95 @@
...
-                <h2>Woanders</h2>
-                <p>Wenn du nicht im Publikum sitzt, sind die Video- und Audio-Streams
-                  des VOC die einfachste Lösung. Im Tab <b>Formate</b> findest du eine Auswahl von übersetzten Streams.</p>
+              <section data-lang="de">
+                <h2>Livestreams und Aufnahmen</h2>
+                <p>Wenn du nicht im Publikum sitzt, sind die Video- und Audio-Streams des VOC die einfachste Lösung. Im Tab <b>Formate</b> findest du eine Auswahl von übersetzten Streams.</p>
+              </section>
+              <section data-lang="fr">
+                <h2>Livestreams et Enregistrements</h2>
+                <p>Si tu n'est pas dans le public, les streams audio et video du VOC la solution la plus simple. Tu trouveras un choix de streams traduits dans la languette <b>Formats</b>.</p>
```
Comment (pbock) [inline static/index.html:275]:
```suggestion
                <p>Si tu n'es pas dans le public, les streams audio et video du VOC sont la solution la plus simple. Tu trouveras un choix de streams traduits dans la languette <b>Formats</b>.</p>
```

File: static/index.html [hunk 2]
```diff
@@ -232,117 +232,95 @@
...
+                <h2>The c3lingo Wiki</h2>
+                <p>All about our work can be found in our wiki. Want to get started translating with us or want to know more about our work? Have a look here!</p>
+              </section>
+              <section data-lang="de">
+                <h2>Das c3lingo-Wiki</h2>
+                <p>Alles über unsere Arbeit kann in unserem Wiki gefunden werden. Du möchtest auch mal übersetzten ausprobieren oder mehr über unsere Arbeit wissen? Dann guck hier rein!</p>
+              </section>
+              <section data-lang="fr">
+                <h2>Le Wiki c3lingo</h2>
+                <p>Tout sur notre travail peut être trouvé dans notre wiki. Vous voulez commencer à traduire avec nous ou en savoir plus sur notre travail? Jetez un coup d'œil ici!</p>
```
Comment (pbock) [inline static/index.html:290]:
```suggestion
                <p>Tout sur notre travail se trouve dans notre wiki. Tu as envie de commencer à traduire avec nous ou en savoir plus sur notre travail&nbsp;? C'est par ici&nbsp;!</p>
```

File: static/index.html [hunk 3]
```diff
@@ -232,117 +232,95 @@
...
+                <h2>Speakers' File Drop</h2>
+                <p>If you're a speaker we always appreciate getting slides, or better even notes, for us to prepare before the talk.</p>
+              </section>
+              <section data-lang="de">
+                <h2>Sprecher-Dateiablage</h2>
+                <p>Wenn du einen Vortrag hältst sind wir immer für Folien oder sogar Notizen Dankbar, mit denen wir uns auf den Vortrag vorbereiten können.</p>
+              </section>
+              <section data-lang="fr">
+                <h2>Dépôt du Fichier des Orateurs</h2>
+                <p>Si vous êtes un orateur, nous apprécions toujours de recevoir des diapositives, ou mieux encore des notes, pour nous permettre de nous préparer avant l'exposé.</p>
```
Comment (pbock) [inline static/index.html:305]:
```suggestion
                <h2>Dépôt de fichiers pour les conférenciers</h2>
                <p>Si tu es conférencière ou conférencier, nous apprécions toujours de recevoir des diapositives, voire des notes, pour nous permettre de mieux nous préparer avant ton exposé.</p>
```

File: static/index.html [hunk 4]
```diff
@@ -232,117 +232,95 @@
...
-                <p>Si tu n'est pas dans le public, les streams audio et video du VOC la solution la plus simple. Tu trouveras un choix de streams traduits dans la languette <b>Formats</b>.</p>
+
+            <a class="block-link" href="https://speakers.c3lingo.org/">
+              <section data-lang="en">
+                <h2>Speakers' File Drop</h2>
+                <p>If you're a speaker we always appreciate getting slides, or better even notes, for us to prepare before the talk.</p>
+              </section>
+              <section data-lang="de">
+                <h2>Sprecher-Dateiablage</h2>
+                <p>Wenn du einen Vortrag hältst sind wir immer für Folien oder sogar Notizen Dankbar, mit denen wir uns auf den Vortrag vorbereiten können.</p>
```
Comment (pbock) [inline static/index.html:301]:
```suggestion
                <p>Wenn du einen Vortrag hältst, sind wir immer für Folien oder sogar Notizen dankbar, mit denen wir uns auf den Vortrag vorbereiten können.</p>
```

**Review by pbock — CHANGES_REQUESTED** [review]
Good idea on restructuring the code, thanks for this! I’ve made some changes to the French translation to the best of my ability, but it’d be great to get a native speaker to have a look at it.

**Review by pbock — APPROVED** [review]

**Comment by GladOSkar** [comment]
Perfect, thanks! I'll ask for a language review now

**Comment by pbock** [comment]
Give me five minutes to merge my own changes, I’m struggling with GitHub’s code review system

**Comment by GladOSkar** [comment]
Sure

**Comment by pbock** [comment]
All right, there we go

**Comment by GladOSkar** [comment]
I don't want to be pushy but can we get this deployed soon? I think it would be great for visitors to be directed to the right addresses soon. Thanks!

**Comment by pbock** [comment]
Yup, it’s deployed now. I’m sorry I made you wait for so long. 😕

As you may have noticed, I’ve moved the repository to the c3lingo organisation. I’m also working on migrating to GitHub Pages right now to cut down on the bureaucracy.

**Comment by GladOSkar** [comment]
Awesome, thanks!

**Pull request merged**